### PR TITLE
Run test case in CI

### PR DESCRIPTION
**Pull request by dschwoerer**
Includes #1

### CODE REVIEW

File: emc-lite/emc-lite.cxx [hunk 1]
```diff
@@ -45,14 +49,16 @@ class emclite : public PhysicsModel {
...
     kappa_epar = kappa_e0 * pow(Te, 5 / 2);
 
-    bool parallel_sheaths = options["parallel_sheaths"].doc("parallel boundary conditions").withDefault(false);
+    parallel_sheaths = options["parallel_sheaths"]
+                           .doc("parallel boundary conditions")
+                           .withDefault(false);
     Nt = options["Nt"].doc("Density at target").withDefault(1.0);
     Tt = options["Tt"].doc("Temperature at target").withDefault(20.0);
-    Cs = options["Cs"].doc("Sound speed").withDefault(1.0);
+    Cs0 = options["Cs"].doc("Sound speed").withDefault(1.0);
```
Comment (bshanahan) [inline emc-lite/emc-lite.cxx]: perhaps we keep the input consistent with the code? Input should then read "Cs0".

**Review by bshanahan — APPROVED** [review]

**Comment by bshanahan** [comment]
Fixed the Cs0 thing but for some reason cl fails.

**Comment by bshanahan** [comment]
Checks worked before, can't see why my change would make it fail. Merging.

**Comment by dschwoerer** [comment]
Thanks. That seems to be a bug in the GHA to checkout the other branch.

The test run itself worked: https://github.com/dschwoerer/bsting_models/actions/runs/2262744859

**Pull request merged**